.github/workflows/test-runner-on-perf.yml

```diff
@@ -2,6 +2,7 @@ name: Update Perf Env and Run Nightly Test Runner Tests
 env:
   GOOGLE_APPLICATION_CREDENTIALS: /tmp/jade-dev-account.json
   GOOGLE_CLOUD_PROJECT: broad-jade-perf
+  CLOUDSDK_CORE_PROJECT: broad-jade-perf
```

Comment:
I may be late to the conversation, but why set both `CLOUDSDK_CORE_PROJECT` and `gcloud config set project`? I think the env var one will override the `gcloud config` one, but why set both if they're both the same value?

Reply:
So just to clarify -- is `gcloud config set project` broken?

Reply:
Short answer: no. 
Your comment made me think of one more combination. Reordering the commands so that setting the project , then the zone did the trick.